code/modules/mob/living/carbon/life.dm

```diff
@@ -225,6 +226,28 @@
 /mob/living/carbon/proc/handle_changeling()
 	return
 
+/mob/living/carbon/proc/handle_disgust()
+	if(disgust)
+		if(disgust >= DISGUST_LEVEL_GROSS)
+			if(prob(10))
+				stuttering += 1
+				confused += 2
```

Comment:
Does this enable you to dart people with food/reagents they hate... thus causing them to be confused for awhile and unable to run away?

Reply:
Disgust is adjusted when a carbon mob eats food.